fe/fe-core/src/main/java/com/starrocks/catalog/DefaultExpr.java

```diff
@@ -18,4 +28,16 @@ public String getExpr() {
     public void setExpr(String expr) {
         this.expr = expr;
     }
+
+    public Expr obtainExpr() {
+        if (SUPPORTED_DEFAULT_FNS.contains(expr)) {
+            String functionName = expr.replace("()", "");
```

Comment:
If you need to support functions not in the form of xx(), but fun(A, B) in the future, how do you need to be compatible?

Reply:
This can be changed when there is such a demand, so far there is no such demand.